### PR TITLE
Hibernate OGM 5.3.1.Final is out

### PR DESCRIPTION
**Pull request by fax4ever**
Update Hibernate blog announcing the new release: Hibernate OGM 5.3.1.Final

### CODE REVIEW

File: posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc [hunk 1]
```diff
@@ -0,0 +1,118 @@
+= "Hibernate OGM 5.3.1.Final is out"
+Fabio Massimo Ercoli
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We're happy to announce the release of:
+http://hibernate.org/ogm/releases/5.3/#get-it[Hibernate OGM 5.3.1.Final]
```
Comment (gsmet) [inline posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc]: Drop the : , it's useless.

An additional sentence in the preamble might be nice. Something like `This is the first maintenance release of the 5.3 branch.`

File: posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+== What's new compared to 5.3.0.Final?
+
+This ia a fix release for native queries pagination and projection issues.
```
Comment (gsmet) [inline posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc]: s/ia/is/
s/fix/maintenance/
something like:
`This is a maintenance release fixing issues related to native queries pagination and projection.`

File: posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+Now pagination works perfectly for MongoDB native queries that use aggregates.
+You can see a complete list of the changes on the
```
Comment (gsmet) [inline posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc]: s/on the/in the/

File: posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+This version is fully compatible with 5.3.0.
+
+== Components upgrade
+
+ * Hibernate ORM 5.2.16.Final
+
+== Projection and addEntity are not allowed in the same query
+
+It is not anymore possible to execute a native query with projection
```
Comment (gsmet) [inline posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc]: It is not anymore possible -> It is not possible anymore

File: posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+====
+[source,java]
+----
+session.createNativeQuery( "db.Movie.find( {}, { 'id' : 1, 'name' : 1  } )" )
+	.addEntity( Movie.class )
+	.uniqueResult();
+----
+====
+
+An exception will be thrown and this message will be presented on log:
```
Comment (gsmet) [inline posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc]: will be presented on log -> will be logged

File: posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+====
+OGM000091: Projection and addEntity are not allowed in the same query on <Movie>
+====
+
+It is legal instead using **projection** without **addEntity**:
```
Comment (gsmet) [inline posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc]: It is allowed either to use **projection** without **addEntity**:

File: posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+====
+[source,java]
+----
+session.createNativeQuery( "db.Movie.find( {}, { 'id' : 1, 'name' : 1  } )" )
+	.uniqueResult();
+----
+====
+
+Or using **addEntity** without **projection** :
```
Comment (gsmet) [inline posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc]: Or to use...

File: posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+[source,java]
+----
+session.createNativeQuery( "db.Movie.find( {} )" )
+	.addEntity( Movie.class )
+	.uniqueResult();
+----
+====
+
+This is valid for all dialects that support native queries,
+at the moment of writing those are MongoDB and Neo4j dialects.
```
Comment (gsmet) [inline posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc]: This is true for all dialects supporting native queries, which at the moment of writing those lines are the MongoDB and Neo4j dialects.

File: posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+This is moreover the same behaviour of Hibernate ORM SQL dialects and JPQL queries.
```
Comment (gsmet) [inline posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc]: This is consistent with the behavior of Hibernate ORM.

File: posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+Indeed if I try to execute a JPQL query using projection a row type result, instead of list of entities, are returned.
```
Comment (gsmet) [inline posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc]: This is useless, just drop it.

**Comment by fax4ever** [comment]
Thank you @gsmet for the review. So it's much much better now thanks.

**Comment by fax4ever** [comment]
Thank you I merged

**Pull request closed without merging**